packaging/brew/ignite.rb

```diff
@@ -0,0 +1,20 @@
+class Ignite < Formula
```

Comment:
Does this file need to be updated for each release?

Reply:
Hi, no it doesn't, this is just for saving it here too, but the file being updated will be in https://github.com/Homebrew/homebrew-core repo.